### DISCUSSION TITLE
Filtering options on opportunity overview page

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#399] [Ideas] [answered]
I would love to be able to quickly filter opportunities via tags. For that it would helpful if all tags are visible on the opportunity overview page so you can just click them to filter (see very rough sketch below).

![Screenshot 2021-08-05 12 03 15](https://user-images.githubusercontent.com/87815729/128332790-472ae71f-5271-40a5-b523-47142b7ddf5d.png)

**Comment by techsmyth** [comment, marked as answer]
Great suggestion! Will be a nice addition. 

Captured on the backlog: https://github.com/alkem-io/client-web/issues/949

Note: also want the same on the ecoverse level for filtering challenges.
